package/MDAnalysis/core/groups.py

```diff
@@ -20,11 +21,12 @@
 import numpy as np
 import functools
 import itertools
+import os
```

Comment:
No the same goes for transformations. I'm writing tests now that are affected by the changes. But this just shows how important a good style checker is.